include/ddc/kernels/splines/matrix.hpp

```diff
@@ -69,6 +108,15 @@ class Matrix
         return b;
     }
 
+    /**
+     * @brief Solve the multiple-right-hand-side linear problem Ax=b inplace.
+     *
+     * @param[in, out] bx A 2D mdpsan storing the multiple right-hand-sides of the problem and receiving the corresponding solution.
+     * Important note: the convention is the reverse of the common matrix one, row number is second index and column number
+     * the first one. It should be changed in the future.
```

Comment:
```suggestion
     * Important note: the convention is the reverse of the common matrix one, row number is second index and column number
     * the first one. This means when solving `A x_i = b_i`,  element `(b_i)_j` is stored in `b(j, i)`.
```

Did I understand correctly ?

Reply:
yes